### PR TITLE
Polish translation.

### PR DESCRIPTION
**Pull request by Aenye-Cerbin**

### CODE REVIEW

**Comment by gyunaev** [comment]
You got an error in your file - build fails with:

Parse error at /home/runner/work/birdtray/birdtray/src/translations/main_pl.ts:389:21: Expected '>' or '/', but got '"'.

**Review by Abestanis — CHANGES_REQUESTED** [review]
Hi, thank you for this translation and for opting into keep them updated in the future. ❤️

We know that writing translations is not an easy thing to to, so we want to make it as easy as possible for you. Unfortunately, there are a few problems with your translation at the moment, which we need to fix before we can merge your changes.

The main problem is your use of the tab character (`\t`). In your editor it might look like the same as four spaces, but they are not the same and here on GitHub you can see the difference:
Tabs used in this pull request
Please go trough your changes and make sure to use spaces instead of the tab character.

For some of the other problems I left some comments further down at the places in the translation files.

I also just recently created a tool to automatically check for problems in translation files. Maybe you can have a look at the warnings it emitted and see if the translations need to be changed.

<details>
  <summary>Warnings emitted by the tool</summary>
  
```
Warning (file=src\translations\installer_es.ts,line=1,col=1): Missing language name "Lang_tr"
Warning (file=src\translations\installer_tr.ts,line=1,col=1): Missing language name "Lang_pl"
Processing src/translations\main_pl.ts...
Warning (file=src/translations\main_pl.ts,line=15,col=61): Source ends with a newline but translation does not (newline_end_src_diff)
Warning (file=src/translations\main_pl.ts,line=15,col=60): Translation ends with different punctuation than the source: Expected "
", got ":" (punctuation_end_differ)
Warning (file=src/translations\main_pl.ts,line=20,col=60): Source ends with a newline but translation does not (newline_end_src_diff)
Warning (file=src/translations\main_pl.ts,line=20,col=59): Translation ends with different punctuation than the source: Expected "
", got ":" (punctuation_end_differ)
Warning (file=src/translations\main_pl.ts,line=24,col=61): Translation ends with different punctuation than the source: Expected "d", got "." (punctuation_end_differ)
Warning (file=src/translations\main_pl.ts,line=28,col=51): Source ends with a whitespace but translation does not (whitespace_end_src_diff)
Warning (file=src/translations\main_pl.ts,line=28,col=50): Translation ends with different punctuation than the source: Expected " ", got ":" (punctuation_end_differ)
Warning (file=src/translations\main_pl.ts,line=55,col=51): Translation ends with different punctuation than the source: Expected ".", got "7" (punctuation_end_differ)
Warning (file=src/translations\main_pl.ts,line=63,col=37): Translation ends with different punctuation than the source: Expected ".", got "a" (punctuation_end_differ)
Warning (file=src/translations\main_pl.ts,line=67,col=49): Translation ends with different punctuation than the source: Expected ".", got "e" (punctuation_end_differ)
Warning (file=src/translations\main_pl.ts,line=152,col=67): Translation ends with different punctuation than the source: Expected ":", got "h" (punctuation_end_differ)
Warning (file=src/translations\main_pl.ts,line=172,col=48): Translation ends with different punctuation than the source: Expected ":", got "ń" (punctuation_end_differ)
Warning (file=src/translations\main_pl.ts,line=188,col=32): Translation ends with different punctuation than the source: Expected ":", got "e" (punctuation_end_differ)
Warning (file=src/translations\main_pl.ts,line=244,col=22): Source starts with a whitespace but translation does not (whitespace_start_src_diff)
Warning (file=src/translations\main_pl.ts,line=308,col=47): Translation ends with a whitespace but source does not (whitespace_end_tr_diff)
Warning (file=src/translations\main_pl.ts,line=337,col=67): Source ends with a newline but translation does not (newline_end_src_diff)
Warning (file=src/translations\main_pl.ts,line=417,col=41): Translation ends with different punctuation than the source: Expected "s", got ":" (punctuation_end_differ)
Warning (file=src/translations\main_pl.ts,line=485,col=91): Translation ends with different punctuation than the source: Expected "s", got "." (punctuation_end_differ)
Warning (file=src/translations\main_pl.ts,line=501,col=22): The translation is missing at least one newline (newline_missing_in_translation)
Warning (file=src/translations\main_pl.ts,line=521,col=60): Translation ends with different punctuation than the source: Expected ".", got "ć" (punctuation_end_differ)
Warning (file=src/translations\main_pl.ts,line=576,col=40): Translation ends with different punctuation than the source: Expected ".", got "a" (punctuation_end_differ)
Warning (file=src/translations\main_pl.ts,line=584,col=45): Translation ends with different punctuation than the source: Expected ".", got "j" (punctuation_end_differ)
Warning (file=src/translations\main_pl.ts,line=588,col=54): Translation ends with different punctuation than the source: Expected ".", got ")" (punctuation_end_differ)
Warning (file=src/translations\main_pl.ts,line=596,col=33): Translation ends with different punctuation than the source: Expected ".", got "u" (punctuation_end_differ)
Error (file=src/translations\main_pl.ts,line=692,col=22): The following text was expected to be present in the translation, but was not found: "1" (special_pattern_missing)
Warning (file=src/translations\main_pl.ts,line=755,col=51): Translation ends with different punctuation than the source: Expected ".", got "1" (punctuation_end_differ)
Warning (file=src/translations\main_pl.ts,line=770,col=35): Translation ends with different punctuation than the source: Expected ":", got "a" (punctuation_end_differ)
Warning (file=src/translations\main_pl.ts,line=809,col=53): Translation ends with different punctuation than the source: Expected ".", got "y" (punctuation_end_differ)
Warning (file=src/translations\main_pl.ts,line=817,col=52): Translation ends with different punctuation than the source: Expected ".", got "y" (punctuation_end_differ)
Found 28 warning(s) and 1 error(s)
Processing src/translations\installer_pl.ts...
Warning (file=src/translations\installer_pl.ts,line=33,col=59): Translation ends with different punctuation than the source: Expected "k", got "." (punctuation_end_differ)
Error (file=src/translations\installer_pl.ts,line=185,col=22): The following text was expected to be present in the translation, but was not found: "${PRODUCT_NAME}" (special_pattern_missing)
Warning (file=src/translations\installer_pl.ts,line=255,col=2): The start of the "message" element has an unexpected indentation: Expected "    ", got "\t" (indentation_wrong_start)
Warning (file=src/translations\installer_pl.ts,line=259,col=2): The start of the "message" element has an unexpected indentation: Expected "    ", got "\t" (indentation_wrong_start)
Warning (file=src/translations\installer_pl.ts,line=260,col=3): The start of the "source" element has an unexpected indentation: Expected "        ", got "\t\t" (indentation_wrong_start)
Warning (file=src/translations\installer_pl.ts,line=261,col=3): The start of the "translation" element has an unexpected indentation: Expected "        ", got "\t\t" (indentation_wrong_start)
Warning (file=src/translations\installer_pl.ts,line=262,col=2): The end of the "message" element has an unexpected indentation: Expected "    ", got "\t" (indentation_wrong_end)
Found 6 warning(s) and 1 error(s)
```
</details>

File: src/translations/installer_pl.ts [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+        <source>MissingVcRuntime_UnableToRun</source>
+        <translation>${PRODUCT_NAME} nie będzie możliwy do uruchomienia.</translation>
+    </message>
+    <message>
+        <source>MissingVcRuntime_UnableToRunDialog</source>
+        <translation>${PRODUCT_NAME} nie może zostać uruchomiony z powodu braku wymaganego komponentu.</translation>
+    </message>
+    <message>
+        <source>Lang_en</source>
+        <translation>angielski</translation>
```
[the remark this answers is not in the document]
Comment (Aenye-Cerbin) [reply src/translations/installer_pl.ts:225]: Yes, in Polish it should be written in lowercase.

**Comment by Aenye-Cerbin** [comment]
@Abestanis 
Thanks for help.
I've tried to replace tabs with spaces (4), but checks are still failing.
Unfortunately I can't get the log.
I'm new to git and github.

**Review by Abestanis — CHANGES_REQUESTED** [review]
> Unfortunately I can't get the log.
I'm new to git and github.

Don't worry, I found it confusing at first too. But thank you for putting in the time an effort, I promise it will get easier every time 😄.

There are two ways to see the errors:
1. If you click on the `Files changed` button you can see the list files and their changes that you are proposing.
Files changed
   here you can see detected errors
Error in translation
  and some of the warnings.
Warning in translation
  Unfortunately, not all of the warnings are shown, GitHub seems to only show a fixed maximum of warnings this way. If you want to see all the warnings, you need to use the second option.
2. At the bottom of your pull request, you should see the following box:
Pull request checks overview box
  You can see that two checks are failing, but please ignore the `CodeQL / Analyze (cpp) (pull_request)` check, it is not finished yet. But the `Check the translations / check (pull_request) ` check is the important one for you. You can see that it failed. If you click on details, you will see the complete log of the failed check.
Failed check log
  Unfortunately, it looks like GitHub does not display the line number of the warnings and errors, But you might deduce from the message where the error or warning is. Also all of them are ordered from the top down, so you can search the next warning somewhere below the previous.

Please also note that there are warnings for other translation files that are not added by you (mostly unfinished translation warnings). You can safely ignore those. Focus on the warnings in the files you added or changed.

However, there is a `Lang_pl` translation missing in `src/translations/installer_tr.ts`. Could you please add that?

To help you, I left suggestions for each of the problems found by the tool. Please have a look at the suggestions and the warnings as soon as possible. We want to create a new release soon and I would like to see your translation included in the next release.

File: src/translations/main_pl.ts [hunk 2]
```diff
@@ -0,0 +1,833 @@
...
+    <message>
+        <source>Failed to save the Birdtray installer:
+</source>
+        <translation>Nie można zapisać instalatora Birdtray:
+</translation>
+    </message>
+    <message>
+        <source>Failed to download the Birdtray installer:
+</source>
+        <translation>Nie można pobrać instalatora Birdtray:</translation>
```
Comment (Abestanis) [inline src/translations/main_pl.ts:21]:
```suggestion
        <translation>Nie można pobrać instalatora Birdtray:
</translation>
```

File: src/translations/main_pl.ts [hunk 3]
```diff
@@ -0,0 +1,833 @@
...
+        <source>10 minutes</source>
+        <translation>10 minut</translation>
+    </message>
+    <message>
+        <source>30 minutes</source>
+        <translation>30 minut</translation>
+    </message>
+    <message>
+        <source>1 hour</source>
+        <translation>godzina</translation>
```
[the remark this answers is not in the document]
Comment (Aenye-Cerbin) [reply src/translations/main_pl.ts]: Ok, I've added the missing 1.
BTW, there is no difference in Polish if You add 1 or not.
For more then 1 the suffix will be different: 2 hours - 2 godziny.

**Comment by Aenye-Cerbin** [comment]
@Abestanis 
Thanks for help. I've added missing lang_pl in installer_tr.ts.
Unfortunately I can't get the log in check details.
That's what I see:
tmc

File: src/translations/main_pl.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (Abestanis) [reply src/translations/main_pl.ts]: Ok, as I said you could have not added the `1` and add the `translatorcomment` instead that I mentioned above.

**Comment by Abestanis** [comment]
Thank you for the changes. There are still some warnings and some comments, could you address them as well?
I think you missed them, because you need to click on this button to see them.
Show hidden conversations button

File: CMakeLists.txt [hunk 4]
```diff
@@ -138,7 +139,8 @@ if(Qt5LinguistTools_FOUND)
             src/translations/dynamic_pt.ts
             src/translations/dynamic_es.ts
             src/translations/dynamic_sv.ts
-            src/translations/dynamic_tr.ts
+            src/translations/main_tr.ts
```
Comment (Abestanis) [inline CMakeLists.txt:142]:
```suggestion
            src/translations/dynamic_tr.ts
```

**Review by Abestanis — APPROVED** [review]
There is one small problem with the `CMakeList.txt` file, but I can fix that for you. Again, thank you very much for the translation and  staying active during the review process!

**Pull request merged**